docs/Contributing/API-Versioning.md

```diff
@@ -1,5 +1,9 @@
 # API versioning
 
+All of the Fleet API routes currently support `v1` in the URL path.
+
+When Fleet 5 is released, the Fleet API will use a new versioning (specified below) and support for `v1` will be removed.
```

Comment:
Maybe change `a new versioning` to `date versioning`

Reply:
@lukeheath good idea. Makes sense. My latest commit adds this.